src/MOI_wrapper/objectives.jl

```diff
@@ -1,14 +1,40 @@
-# TODO: MOI.supports(::Optimizer, ::MOI.ObjectiveSense) = true
-MOI.supports(::Optimizer, ::MOI.ObjectiveFunction{ScalarFunction{F}}) where {F <: Function} = true
+MOI.supports(::Optimizer, ::MOI.ObjectiveSense) = true
+MOI.get(model::Optimizer, ::MOI.ObjectiveSense) = Min
```

Comment:
Did you define `Min`? It should be `MOI.MIN_SENSE`.

Reply:
Good catch! Thanks